app/models/proof_attempt.rb

```diff
@@ -1,6 +1,13 @@
 class ProofAttempt < ActiveRecord::Base
   belongs_to :theorem, foreign_key: 'sentence_id'
   belongs_to :proof_status
+  has_and_belongs_to_many :used_axioms,
+    class_name: 'Sentence',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.